overlord/snapstate/snapmgr_test.go

```diff
@@ -641,7 +642,7 @@ func (s *snapmgrTestSuite) TestInstallRunThrough(c *C) {
 		},
 		{
 			op:   "copy-data",
-			name: "/snap/some-snap/11",
+			name: fmt.Sprintf("%s/some-snap/11", dirs.SnapMountDir),
```

Comment:
@niemeyer I would disagree and suggest that having it not hardcoded anywhere and then being able to create mock tests where you try different paths and everything works is important. Snaps are supposed to not care about where they exist on the host filesystem, and this is a good way to prove it.